doc/tools/docgram/fullGrammar

```diff
@@ -2471,14 +2524,22 @@ rewriter: [
 | natural [ "?" | LEFTQMARK ] constr_with_bindings_arg
 | natural constr_with_bindings_arg
 | constr_with_bindings_arg
+| "!" constr_with_bindings
+| [ "?" | LEFTQMARK ] constr_with_bindings
+| lnatural "!" constr_with_bindings
+| lnatural [ "?" | LEFTQMARK ] constr_with_bindings
+| lnatural constr_with_bindings
+| constr_with_bindings
 ]
 
 oriented_rewriter: [
 | orient_rw rewriter
+| ltac2_orient rewriter
 ]
 
 induction_clause: [
 | destruction_arg as_or_and_ipat eqn_ipat opt_clause
+| destruction_arg as_or_and_ipat eqn_ipat OPT clause
```

Comment:
There are some grammar rules that are both in the Ltac1 and Ltac2 plugin but which are grammar rules for tactics themselves, so independent of which version of Ltac is used. It would be interesting to figure out if it is possible to factorize them, but I'm not so hopeful.